src/configurations/destinations/accoil_analytics/db-config.json

```diff
@@ -0,0 +1,121 @@
+{
+  "name": "Accoil_Analytics",
+  "displayName": "Accoil Analytics",
+  "config": {
+    "cdkV2Enabled": true,
+    "transformAtV1": "processor",
+    "saveDestinationResponse": true,
+    "includeKeys": [
+      "oneTrustCookieCategories",
+      "connectionMode",
+      "consentManagement",
+      "ketchConsentPurposes"
+    ],
+    "excludeKeys": [],
+    "supportedSourceTypes": [
+      "android",
+      "ios",
+      "web",
+      "unity",
+      "amp",
+      "cloud",
+      "warehouse",
+      "reactnative",
+      "flutter",
+      "cordova",
+      "shopify"
+    ],
+    "supportedMessageTypes": {
+      "cloud": ["identify", "page", "screen", "track", "group"]
+    },
+    "supportedConnectionModes": {
+      "web": ["cloud"],
+      "android": ["cloud"],
+      "ios": ["cloud"],
+      "unity": ["cloud"],
+      "amp": ["cloud"],
+      "reactnative": ["cloud"],
+      "flutter": ["cloud"],
+      "cordova": ["cloud"],
+      "shopify": ["cloud"],
+      "cloud": ["cloud"],
+      "warehouse": ["cloud"]
+    },
+    "destConfig": {
+      "defaultConfig": ["apiKey", "blacklistedEvents", "whitelistedEvents", "eventFilteringOption"],
```

Comment:
why are we including other keys? I see only apiKey is in use. We should remove the other keys?

Reply:
Good catch. I've removed the others and left apiKey.